src/eureka/S3_data_reduction/nirspec.py

```diff
@@ -143,6 +138,44 @@ def flag_bg(data, meta, log):
     return data
 
 
+def flag_ff(data, meta, log):
+    '''Outlier rejection of full frame along time axis.
+    For data with deep transits, there is a risk of masking good transit data.
+    Proceed with caution.
```

Comment:
This is currently an exact clone of the nircam.py function, so you should just call that function like you've done in miri.py